pkg/scheduler/algorithm/predicates/error.go

```diff
@@ -30,31 +30,31 @@ var (
-	ErrNodeSelectorNotMatch                  = newPredicateFailureError("MatchNodeSelector")
-	ErrPodAffinityNotMatch                   = newPredicateFailureError("MatchInterPodAffinity")
-	ErrPodAffinityRulesNotMatch              = newPredicateFailureError("PodAffinityRulesNotMatch")
-	ErrPodAntiAffinityRulesNotMatch          = newPredicateFailureError("PodAntiAffinityRulesNotMatch")
-	ErrExistingPodsAntiAffinityRulesNotMatch = newPredicateFailureError("ExistingPodsAntiAffinityRulesNotMatch")
-	ErrTaintsTolerationsNotMatch             = newPredicateFailureError("PodToleratesNodeTaints")
-	ErrPodNotMatchHostName                   = newPredicateFailureError("HostName")
-	ErrPodNotFitsHostPorts                   = newPredicateFailureError("PodFitsHostPorts")
-	ErrNodeLabelPresenceViolated             = newPredicateFailureError("CheckNodeLabelPresence")
-	ErrServiceAffinityViolated               = newPredicateFailureError("CheckServiceAffinity")
-	ErrMaxVolumeCountExceeded                = newPredicateFailureError("MaxVolumeCount")
-	ErrNodeUnderMemoryPressure               = newPredicateFailureError("NodeUnderMemoryPressure")
-	ErrNodeUnderDiskPressure                 = newPredicateFailureError("NodeUnderDiskPressure")
-	ErrNodeOutOfDisk                         = newPredicateFailureError("NodeOutOfDisk")
-	ErrNodeNotReady                          = newPredicateFailureError("NodeNotReady")
-	ErrNodeNetworkUnavailable                = newPredicateFailureError("NodeNetworkUnavailable")
-	ErrNodeUnschedulable                     = newPredicateFailureError("NodeUnschedulable")
-	ErrNodeUnknownCondition                  = newPredicateFailureError("NodeUnknownCondition")
-	ErrVolumeNodeConflict                    = newPredicateFailureError("VolumeNodeAffinityConflict")
-	ErrVolumeBindConflict                    = newPredicateFailureError("VolumeBindingNoMatch")
+	ErrDiskConflict                          = newPredicateFailureError("NoDiskConflict", "Nodes had disk conflict")
+	ErrVolumeZoneConflict                    = newPredicateFailureError("NoVolumeZoneConflict", "Nodes had volumeZone conflict")
+	ErrNodeSelectorNotMatch                  = newPredicateFailureError("MatchNodeSelector", "Nodes didn't match node selector")
+	ErrPodAffinityNotMatch                   = newPredicateFailureError("MatchInterPodAffinity", "Nodes didn't match pod affinity/anti-affinity")
+	ErrPodAffinityRulesNotMatch              = newPredicateFailureError("PodAffinityRulesNotMatch", "Nodes didn't match pod affinity rules")
+	ErrPodAntiAffinityRulesNotMatch          = newPredicateFailureError("PodAntiAffinityRulesNotMatch", "Nodes didn't match pod anti-affinity rules")
+	ErrExistingPodsAntiAffinityRulesNotMatch = newPredicateFailureError("ExistingPodsAntiAffinityRulesNotMatch", "Nodes didn't satisfy existing pods anti-affinity rules")
+	ErrTaintsTolerationsNotMatch             = newPredicateFailureError("PodToleratesNodeTaints", "Pod didn't tolerates node taints")
```

Comment:
Since these appear in the unschedulable message with the number of evaluated nodes, they should start with "Nodes...". So, please change this message to "Nodes had taints that the pod didn't tolerate".